### PR TITLE
python3.pkgs.django-shortuuidfield: init at 0.1.3

### PR DESCRIPTION
**Pull request by DerDennisOP**
###### Things done

- Built on platform(s)
  - [x] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- [ ] For non-Linux: Is `sandbox = true` set in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
- [x] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [ ] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [23.05 Release Notes (or backporting 22.11 Release notes)](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#generating-2305-release-notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

File: pkgs/development/python-modules/django-shortuuidfield/default.nix [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+    shortuuid
+    six
+  ];
+
+  pythonImportsCheck = [
+    "shortuuidfield"
+  ];
+
+  meta = with lib; {
+    description = "Short UUIDField for Django. Good for use in urls & file names.";
```
Comment (ocfox) [inline pkgs/development/python-modules/django-shortuuidfield/default.nix:30]:
```suggestion
    description = "Short UUIDField for Django. Good for use in urls & file names";
```

File: pkgs/development/python-modules/django-shortuuidfield/default.nix [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+  src = fetchPypi {
+    inherit pname version;
+    hash = "sha256-opLA/lU4q+lHsTHiuRTt2axEr8xqQOrscUSOYjGj7wA=";
+  };
+
+  propagatedBuildInputs = [
+    django
+    shortuuid
+    six
+  ];
```
Comment (SuperSandro2000) [inline pkgs/development/python-modules/django-shortuuidfield/default.nix:23]:
```suggestion
  ];

  # no tests
  doCheck = false;
```

**Comment by SuperSandro2000** [comment]
@ofborg build python3.pkgs.django-shortuuidfield

**Comment by SuperSandro2000** [comment]
@DerDennisOP please rebase on master and while you are at it, please squash the commits together

**Comment by DerDennisOP** [comment]
I coudn't get the rebase propertly done

**Comment by yayayayaka** [comment]
Seems you were trying to merge your fork's master branch into this one. To rebase against `upstream/master`, do the following:
1. Undo the merge: `git reset --hard HEAD~1`
2. Squash the commits together as Sandro suggested (for example with `git rebase -i HEAD~2`)
3. Rebase against upstream/master and force-push the result onto this branch:
   ```
   git fetch upstream
   git rebase upstream/master
   # Resolve merge conflicts
   git push --force-with-lease
   ```

**Comment by natsukium** [comment]
Result of `nixpkgs-review pr 225789` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>4 packages built:</summary>
  <ul>
    <li>python310Packages.django-shortuuidfield</li>
    <li>python310Packages.django-shortuuidfield.dist</li>
    <li>python311Packages.django-shortuuidfield</li>
    <li>python311Packages.django-shortuuidfield.dist</li>
  </ul>
</details>

**Review by natsukium — APPROVED** [review]
LGTM

I'll squash the commits.

**Comment by natsukium** [comment]
Result of `nixpkgs-review pr 225789` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>4 packages built:</summary>
  <ul>
    <li>python310Packages.django-shortuuidfield</li>
    <li>python310Packages.django-shortuuidfield.dist</li>
    <li>python311Packages.django-shortuuidfield</li>
    <li>python311Packages.django-shortuuidfield.dist</li>
  </ul>
</details>

**Pull request merged**